integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/StandardBusinessDocumentFactory.java

```diff
@@ -125,8 +125,9 @@ public EduDocument create(ConversationResource shipmentMeta, MessageContext cont
         }
         Payload payload = new Payload(encryptArchive(context.getMottaker(), archive));
 
-        return new CreateSBD().createSBD(context.getAvsender().getOrgNummer(), context.getMottaker().getOrgNummer(), payload,
-                context.getConversationId(), StandardBusinessDocumentHeader.NEXTBEST_TYPE, context.getJournalPostId());
+        EduDocument sbd = new CreateSBD().createSBD(context.getAvsender().getOrgNummer(), context.getMottaker().getOrgNummer(), payload,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "sbd". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1488)